<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherHome</class>
 <widget class="QDialog" name="TeacherHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{background-color:rgb(43, 49, 56);
}
QLabel
{
color:rgb(0, 192, 255);
font: 12pt &quot;Segoe Print&quot;;
}
QPushButton
{
border-style:bold;
background-color:rgb(0, 192, 255);
color:white;


border-radius:5px;
}
QPushButton:hover
{
background-color:white;
color:rgb(0, 192, 255);
border-color:green;
border-width:10px;
border-style:bold;
}
QPushButtonpressed
{
color:rgb(255, 255, 0);
background-color:rgb(52, 168, 83);
border-width:2px;}
QRadioButton
{
color:rgb(0, 192, 255);}
QGroupBox
{font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(0,192,255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
</string>
   </property>
   <property name="text">
    <string>&lt;h1&gt;&lt;u&gt;Teacher Home&lt;/u&gt;&lt;/h1&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;

</string>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
</string>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
</string>
   </property>
   <property name="text">
    <string>Mobile Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
</string>
   </property>
   <property name="text">
    <string>Email ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Enroll</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>209</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>209</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>209</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>209</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>209</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Unenroll</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-radius:100px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
